src/ir/module-utils.h

```diff
@@ -132,12 +132,9 @@ inline DataSegment* copyDataSegment(const DataSegment* segment, Module& out) {
   return out.addDataSegment(std::move(ret));
 }
 
-inline void copyModule(const Module& in, Module& out) {
-  // we use names throughout, not raw pointers, so simple copying is fine
-  // for everything *but* expressions
-  for (auto& curr : in.exports) {
-    out.addExport(new Export(*curr));
-  }
+// Copies named toplevel module items (things of kind ModuleItemKind). See
+// copyModule() for something that also copies exports, the start function, etc.
+inline void copyModuleItems(const Module& in, Module& out) {
```

Comment:
Is this ever independently useful? If not, I suggest that we roll it into `copyModule`. Also, these methods seem expensive enough that it would make more sense to put them in a .cpp file rather than in a header.

Reply:
Yes, it's used in both `copyModule` and in `wasm-merge` - maybe I don't understand your question? Are you suggesting to duplicate that code?

I agree we should move this out of headers. I can do that afterwards.

Reply:
Great, I didn't get far enough to see that it's used in two places. Moving the code out of the header as a follow-up sgtm.